armory/scenarios/librispeech_dev_clean.py

```diff
@@ -0,0 +1,86 @@
+"""
+Librispeech Dev Clean scenario evaluation
+"""
+
+import logging
+
+from tqdm import tqdm
+
+from armory.utils.config_loading import (
+    load_dataset,
+    load_model,
+    load_attack,
+)
+from armory.utils import metrics
+from armory.scenarios.base import Scenario
+
+logger = logging.getLogger(__name__)
+
+
+class LibrispeechDevClean(Scenario):
+    def _evaluate(self, config: dict) -> dict:
+        """
+        Evaluate the config and return a results dict
+        """
+
+        model_config = config["model"]
+        classifier, preprocessing_fn = load_model(model_config)
+
+        if not model_config["weights_file"]:
+            classifier.set_learning_phase(True)
+            logger.info(
+                f"Fitting model {model_config['module']}.{model_config['name']}..."
+            )
+            fit_kwargs = model_config["fit_kwargs"]
+            train_data = load_dataset(
+                config["dataset"],
+                epochs=fit_kwargs["nb_epochs"],
+                split_type="train",
+                preprocessing_fn=preprocessing_fn,
+            )
+            classifier.fit_generator(train_data, **fit_kwargs)
+
+        classifier.set_learning_phase(False)
+        # Evaluate the ART classifier on benign test examples
+        logger.info(f"Loading dataset {config['dataset']['name']}...")
+        test_data_generator = load_dataset(
+            config["dataset"],
+            epochs=1,
+            split_type="test",
+            preprocessing_fn=preprocessing_fn,
+        )
+        logger.info("Running inference on benign examples...")
+
+        task_metric = metrics.categorical_accuracy
```

Comment:
Do we want this to be picked up from the config `metric`? or should we remove metric from the configs and have them be coded into the scenario so they're compatible?

Reply:
It would be nice to pick it up from the config. That way you could track different (or additional metrics) as desired.